cruds_rails/template.rb

```diff
@@ -0,0 +1,110 @@
+# Add the current directory to the path Thor uses
+# to look up files
+def source_paths
+  Array(super) +
+    [File.expand_path(File.dirname(__FILE__))]
+end
+
+def curl_file(name)
+  run "curl \"https://raw.githubusercontent.com/CruGlobal/cruprojects/add-template/cruds_rails/#{name}\" -o \"#{name}\""
+end
+
+remove_file 'Gemfile'
+run 'touch Gemfile'
+add_source 'https://rubygems.org'
+gem 'rails', '~> 4.2.1'
+gem 'puma'
+
+gem 'pg'
+
+gem 'newrelic_rpm'
+gem 'rack-cors', require: 'rack/cors'
+gem 'redis', '~> 3.2.1'
+gem 'redis-rails'
+gem 'rollbar'
+gem 'silencer'
+gem 'syslog-logger'
+
+gem_group :development, :test do
+  gem 'dotenv-rails'
+  gem 'guard'
+  gem 'guard-rubocop'
+  gem 'rspec-rails'
+  gem 'spring'
+  gem 'pry-rails'
+end
+
+gem_group :test do
+  gem 'guard-rspec'
```

Comment:
Why is guard-rspec in this group and the other 2 guard things in the other group?